web/src/main/java/org/phoenixctms/ctsms/web/jersey/resource/ArgsUriPart.java

```diff
@@ -20,43 +19,23 @@
 import org.phoenixctms.ctsms.util.AssociationPath;
 import org.phoenixctms.ctsms.util.MethodTransfilter;
 
-public class ArgsUriPart implements UriPart {
+public class ArgsUriPart {
 
 	private final static Comparator<String> PARAMETER_TO_ARGUMENT_SORTING = null;
-	// private Method method;
 	private boolean slurp;
 	private String resource;
 	private LinkedHashMap<String, Object> overrides;
 	private LinkedHashMap<String, Object> defaults;
 	private HashSet<String> excludePrimitiveConversion;
-	private Class declaringInterface;
-	private ArrayList<StringConverter> primitiveConversionPrecedence;
+	private Class<?> declaringInterface;
+	private ArrayList<StringConverter<?>> primitiveConversionPrecedence;
 	private MethodTransfilter methodTransfilter;
 
-	// public ArgsUriPart(Method method) {
-	// this(method, null);
-	// }
-	// return arguments.toArray()
-	// public ArgsUriPart() {
-	// this(null, null, null);
-	// }
-	public ArgsUriPart(Class declaringInterface) {
-		this(declaringInterface, null, MethodTransfilter.DEFAULT_TRANSFILTER);
-	}
-
-	public ArgsUriPart(Class declaringInterface, MethodTransfilter methodTransfilter) {
-		this(declaringInterface, null, methodTransfilter);
-	}
-
-	public ArgsUriPart(Class declaringInterface, String resource) {
-		this(declaringInterface, resource, MethodTransfilter.DEFAULT_TRANSFILTER);
-	}
```

Comment:
ok these are unused constructors.
when introducing a multiargument constructor ie. ArgsUriPart(Class declaringInterface, String resource, MethodTransfilter methodTransfilter), either eclipse source autocreate constructors or me has the habit of creating additional constructors for subsets of arguments.